### ISSUE TITLE
String treatment API

### ISSUE THREAD

**Issue by cosmologicon** [mmachenry/mystery-hunt-sheets-addons#9]
I can think of a variety of string functions we could use and I'm wondering if it makes sense to have a coherent API around exactly how you're treating a string. For example, I think it would be useful to remove all the letters from one string from another:

`removeChars("abcdef", "bf") => "acde"`

But there's a lot of details that aren't obvious. Does order matter? Do multiple instances of a character matter? Do characters that aren't in the first string matter?

Basically I think we usually want to treat strings as one data structure or another vis-a-vis the characters they contain. Sometimes we want them to be list/array, sometimes unordered set, sometimes ordered multiset, etc. I'm wondering if there's some kind of consistent naming we could use that makes it clear what to expect when calling the function. My first thought is something like `removeCharsMultiOrdered`, but maybe there's something better.
